views/js/signup.js

```diff
@@ -1,12 +1,90 @@
 $(document).ready(function () {
   //init side bar
   $('.sidenav').sidenav();
+  M.AutoInit();
```

Comment:
do not need both ` $('.sidenav').sidenav();` and ` M.AutoInit();` checkout here: https://materializecss.com/auto-init.html